rpmlint/checks/BinariesCheck.py

```diff
@@ -85,32 +121,15 @@ def __init__(self, config, output, pkg, path, fname, is_ar, is_shlib):
                 self.output.error_details.update({name: func['description']})
 
         is_debug = path.endswith('.debug')
-        is_archive = path.endswith('.a')
+
+        self.checker = BinaryInfoChecker(path)
```

Comment:
Here you init the content but it should be in the BinaryCheck directly I guess.

Reply:
I don't get this?